acquia_lift.admin.wizard.inc

```diff
@@ -71,7 +71,32 @@ function acquia_lift_personalize_campaign_wizard_variations_alter(&$form, &$form
   if (empty($agent_data->machine_name)) {
     return;
   }
+  $status = personalize_agent_get_status($agent_data->machine_name);
+  $variation_set_handling = isset($agent_data->data['variation_set_handling']) ? $agent_data->data['variation_set_handling'] : ACQUIA_LIFT_DECISION_LOCKSTEP;
+
   $option_sets = personalize_option_set_load_by_agent($agent_data->machine_name);
+  if ($variation_set_handling == ACQUIA_LIFT_DECISION_MULTIVARIATE) {
+    // If this is an MVT then the option sets get saved directly on the testing
+    // agent.
+    $mvt_name = acquia_lift_get_mvt_name_for_personalization($agent_data->machine_name);
+    if ($mvt = personalize_agent_load($mvt_name)) {
+      $option_sets = personalize_option_set_load_by_agent($mvt_name);
+    }
+  }
+
+  $form['variations']['variation_set_handling'] = array(
+    '#type' => 'radios',
+    '#options' => array(
+      ACQUIA_LIFT_DECISION_LOCKSTEP => t('Lock step'),
```

Comment:
It will probably come up at mid-sprint review, but just in case this is really just a reminder to both of us to bring up that these probably need some sort of brief description assigned to them.